package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "install": "yarn run theme-base build:all",
+    "install": "yarn run theme-base build:all && yarn core build && yarn react build",
```

Comment:
ohh I like, now I can just run install and then start up the docs or example next server

Reply:
Exactly right @dbanksdesign – I want the majority of the packages/examples/docs/etc. to "just work" with `git clone` and `yarn install`.

The only exception is really `environments/`, and that's because they require an AWS account.

In the future, we can even avoid that by including `aws-exports.js` – we would only need to monitor those resources for usage/abuse.